cmd/internal/dgather.go

```diff
@@ -332,19 +333,20 @@ func RenderJobBPM(currentJob manifest.Job, jobInstances []manifest.JobInstance,
 				},
 
 				&btg.InstanceInfo{
-					Address:    instance.Address,
-					AZ:         instance.AZ,
-					ID:         instance.ID,
-					Index:      string(instance.Index),
+					Address:    jobInstances[0].Address,
+					AZ:         jobInstances[0].AZ,
+					ID:         jobInstances[0].ID,
+					Index:      string(jobInstances[0].Index),
```

Comment:
The point for this loop is to make sure `bpm.yml` doesn't change based on index, address or AZ.
So we need to render bpm with the parameters for each index, to have a correct comparison.